server/routes/referrals/referralFormPresenter.ts

```diff
@@ -1,118 +1,123 @@
+
+    buildSingleReferralSections(): ReferralFormSectionPresenter[] {
+      const reviewServiceUserInformationSection = this.buildReviewServiceUserInformationSection()
+      const referralDetailsSection = this.buildSingleReferralDetailsSection(reviewServiceUserInformationSection)
+      const checkYourAnswersSection = this.buildCheckYourAnswersSection(referralDetailsSection, false)
+      return [reviewServiceUserInformationSection, referralDetailsSection, checkYourAnswersSection]
+    }
+
+    buildCohortReferralSections() {
+      const reviewServiceUserInformationSection = this.buildReviewServiceUserInformationSection()
+      const selectServiceCategoriesSection = this.buildSelectServiceCategoriesSection(
+        reviewServiceUserInformationSection
+      )
+      const referralDetailsSection = this.buildCohortReferralDetailsSection(selectServiceCategoriesSection)
+      const checkYourAnswersSection = this.buildCheckYourAnswersSection(referralDetailsSection, true)
+      return [
+        reviewServiceUserInformationSection,
+        selectServiceCategoriesSection,
+        referralDetailsSection,
+        checkYourAnswersSection,
+      ]
+    }
+
+    private buildReviewServiceUserInformationSection(): ReferralFormSingleListSectionPresenter {
+      return {
+        type: 'single',
+        title: 'Review service user’s information',
+        number: '1',
+        status: this.calculateStatus(this.sectionValues.reviewServiceUserInformation),
+        tasks: [
+          {
+            title: 'Confirm service user’s personal details',
+            url: 'service-user-details',
+          },
+          {
+            title: 'Service user’s risk information',
+            url: this.calculateTaskUrl('risk-information', this.taskValues.serviceUserDetails),
+          },
+          {
+            title: 'Service user’s needs and requirements',
+            url: this.calculateTaskUrl('needs-and-requirements', this.taskValues.riskInformation),
+          },
+        ],
+      }
+    }
+
+    private buildSelectServiceCategoriesSection(
+      reviewServiceUserInformationSection: ReferralFormSingleListSectionPresenter
+    ): ReferralFormSingleListSectionPresenter {
+      return {
+        type: 'single',
+        title: 'Choose service types',
         number: '2',
         status: this.calculateStatus(
-          this.sectionValues.serviceCategoryReferralDetails,
-          reviewServiceUserInformation.status!
+          this.sectionValues.cohortServiceCategories,
+          reviewServiceUserInformationSection.status
         ),
-        taskListSections: [
+        tasks: [
           {
-            tasks: [
-              {
-                title: `Confirm the relevant sentence for the intervention referral`,
-                url: this.calculateTaskUrl('relevant-sentence', this.taskValues.needsAndRequirements),
-              },
-            ],
+            title: `Select service types for the ${this.intervention.title} referral`,
```

Comment:
i think this should be the contract type not the intervention title, to be consistent with other pages. 

```Select service types for the ${this.intervention.contractType.name} referral```

Reply:
and as mentioned above, i think it should be 'service categories' not types